.github/workflows/test-interactive-input.yaml

```
@@ -13,7 +13,7 @@ jobs:
     steps:
       - name: Example Interactive Inputs Step
         id: interactive-inputs
-        uses: boasihq/interactive-inputs@main
+        uses: boasihq/interactive-inputs@ii-i1-allow-users-to-more-easily-copy-target-value-for-select-multiselect-field-types
```

Will revert back to main for merge

```suggestion
        uses: boasihq/interactive-inputs@main
```